reporters/worker-api-plugin/src/main/java/reporters/WorkerApiTask.java

```diff
@@ -26,13 +26,13 @@ public abstract class WorkerApiTask extends DefaultTask {
 
     @TaskAction
     public void execute() {
-//        getProblems().forNamespace("reporters.worker.api.plugin").reporting(problem -> problem
-//            .id("adhoc-worker-deprecation", "Plugin is deprecated")
-//            .contextualLabel("The 'worker-api-plugin' is deprecated")
-//            .documentedAt("https://github.com/gradle/gradle/README.md")
-//            .severity(Severity.WARNING)
-//            .solution("Please use a more recent plugin version")
-//        );
+        getProblems().getReporter().reporting(problem -> problem
+            .id("adhoc-worker-deprecation", "Plugin is deprecated")
+            .contextualLabel("The 'worker-api-plugin' is deprecated")
+            .documentedAt("https://github.com/gradle/gradle/README.md")
+            .severity(Severity.WARNING)
+            .solution("Please use a more recent plugin version")
+        );
```

Comment:
Hey @hegyibalint, the problem with this change is that you have uncommented the report in the `Task`, whereas it is the report in the `WorkAction` that actually runs inside the worker.

I think that this change is the reason that you saw the problem being reported - I left this commented out code to demonstrate that there is no problem with reporting from the task itself.